packages/playwright/src/index.ts

```diff
@@ -668,7 +661,6 @@ class ArtifactsRecorder {
     await this._stopTracing(context.tracing);
 
     await this._screenshotRecorder.captureTemporary(context);
-    await this._pageSnapshotRecorder.captureTemporary(context);
```

Comment:
I think we should keep capturing page snapshot here, otherwise we'll miss the main `context` fixture because it is closed before the `didFinishTest` is called.